content/docs/2.0/scalers/redis-lists.md

```diff
@@ -53,7 +53,7 @@ You can authenticate by using a password.
 
 Here is an example of how to deploy a scaled object with the `redis` scale trigger which uses `TriggerAuthentication`.
 
-You can also provide the `password` on the `ScaledObject` directly.
+You can also provide the `passwordFromEnv` on the `ScaledObject` directly.
 
 ```yaml
```

Comment:
Sample below is not updated: https://github.com/kedacore/keda-docs/pull/241/files#diff-6bfbb18b82b17a7405561fd1316e3f5bR90